brianlong/www.validators.app
# 182556424_replacement_for_get_confirmed_block

#### What's this PR do?
- Replace script custom methods with gem methods

#### How should this be manually tested?
- Run `rails r add_current_epoch.rb`, check the console and results
- Run `rails r back_fill_epoch_data.rb`, check the console and results
- Run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/182556424)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
